src/tracer.test.js

```diff
@@ -463,6 +464,22 @@ describe('tracer', () => {
     expect(Http.hookHttp).toHaveBeenCalledTimes(1);
   });
 
+  test('trace; auto tag - happy flow ', async () => {
+    const token = TOKEN;
+    const lumigoTracer = require('./index')({ token });
+    expect(spies.warnClient).toHaveBeenCalledTimes(0);
+    expect(spies.setSwitchOff).not.toHaveBeenCalled();
+
+    const { context } = new HandlerInputesBuilder().build();
+
+    const userHandler4 = async (event, context, callback) => {
+      return {};
+    };
+    await lumigoTracer.trace(userHandler4)({ key1: 'aaa' }, context, jest.fn());
+
+    expect(spies.autoTagEvent).toBeCalledWith({ key1: 'aaa' });
```

Comment:
Don't use the mock, query the ExecutionTags API and check if there is a tag

Reply:
Unfortunately this is hard :(